app/src/main/java/io/github/droidkaigi/confsched/fragment/SessionDetailFragment.java

```diff
@@ -64,14 +65,35 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         AppUtil.setTaskDescription(activity, session.title, ContextCompat.getColor(activity, session.category.getVividColorResId()));
     }
 
+    // Receive all viewModel's properties changed.
+    private final Observable.OnPropertyChangedCallback onPropertyChanged = new Observable.OnPropertyChangedCallback() {
```

Comment:
> Let's consider about ViewModel lifecycle.
> It should be same as view lifecycle.

I usually make ViewModel interface as follows:

```
public interface ViewModelInterface {
    void onCreate(Bundle savedInstanceState);
    void onResume();
    void onPause();
    void onDestroy();
    void onSaveInstanceState(Bundle outState);
    void onRestoreInstanceState(Bundle savedInstanceState);
}
```

and view delegates own lifecycle callback to ViewModel.
It helps move view's logic to model.